week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -18,36 +19,61 @@ This means the order the songs are played in will be random, but each song will
 
  */
 
-
 class MusicPlayer {
-    // Add your code here
-
+  constructor() {
+    this.playlist = [];
+    this.index = 0;
+  }
+  add(title, artist) {
+    this.playlist.push({ title, artist });
+  }
+
+  play() {
+    if (this.playlist.length === 0) {
+      console.log("Your playlist is empty. Add new songs!");
+    } else {
+      console.log(
+        `Currently playing: ${this.playlist[this.index].title} by ${
+          this.playlist[this.index].artist
+        }`
+      );
+    }
+  }
+
+  skip() {
+    this.index++;
+    if (this.index <= this.playlist.length - 1) {
+      console.log(
+        `Currently playing: ${this.playlist[this.index].title} by ${
```

Comment:
you already have a function that does this, you could call your play function instead of typing out the whole console log again